.github/workflows/release.yml

```diff
@@ -143,3 +143,28 @@ jobs:
           files: release/*
           token: ${{ secrets.GITHUB_TOKEN }}
           name: Version ${{ env.TAG }}
+
+
+  deploy:
+    name: Deploy to abacus-test.nl
```

Comment:
With build-lint-test doing this for the `main` branch, what's the reason for also including it here?

Reply:
The release build is a bit different, its build in release mode, the binary is smaller (for linux) and compiled against musl. We want to test this binary with abacus-test.nl

Reply:
Oh of course, download step is the same, but the build job that does the upload is different between the two workflows.